File: STRIVER/video/vid-5/pattern14/p14.cpp
// A
// AB
// ABC
// ABCD
// ABCDE

#include <iostream>
using namespace std;

int main()
{
    int row = 4;

    for (int i = 0; i <= row; i++)
    { 
        for (char j = 'A'; j<= 'A'+ i; j++){
            cout<<j;
        }
        cout<<endl;
    }

    return 0;
}

File: STRIVER/video/vid-5 (Patterns)/pattern14/p14.py
# A
# AB
# ABC
# ABCD
# ABCDE

def p13(row:int)->None:

    for i in range(row+1):
        for j in range(ord('A'), ord('A') + i + 1):
            print(chr(j),end="")
        print("")

row = 4
p13(row)